bin/mocha

```diff
@@ -118,32 +115,41 @@ if (nodeArgs.gc) {
   delete nodeArgs.gc;
 }
 
-debug('final node args', nodeArgs);
+if (Object.keys(nodeArgs).length) {
+  const {spawn} = require('child_process');
+  const mochaPath = require.resolve('../lib/cli/cli.js');
```

Comment:
For my understanding:
now we spawn "lib/cli/cli.js" instead of "bin/_mocha". Right?
Both ways do fine and pass our `npm test`.

Reply:
yes. I would prefer to "standardize" on `lib/cli/cli.js` and replace all usages of `bin/_mocha` with this, essentially